app/subscriber/src/components/content-list/ContentList.tsx

```diff
@@ -65,104 +64,80 @@ export const ContentList: React.FC<IContentListProps> = ({
 
   return (
     <styled.ContentList>
-      {Object.keys(grouped).map((group) => (
-        <div key={group}>
-          <h2 className="group-title">{group}</h2>
-          <div>
-            {grouped[group].map((item) => (
-              <Col
-                className={`content-row ${
-                  selected.some((selectedItem) => selectedItem.id === item.id) ? 'checked' : ''
-                }`}
-                key={item.id}
-                onClick={(e) => {
-                  // Ensure the target is an Element and use .closest to check if the click was inside a checkbox (see comment below)
-                  if (!(e.target instanceof Element) || !e.target.closest('.checkbox')) {
-                    navigate(`/view/${item.id}`);
-                  }
-                }}
-              >
-                <Row className="parent-row">
-                  <Checkbox
-                    className="checkbox"
-                    checked={selected.some((selectedItem) => selectedItem.id === item.id)}
-                    onChange={(e) => {
-                      // TODO
-                      // e.stopPropagation() does not work, so above we check if the click was inside a checkbox
-                      handleCheckboxChange(item, e.target.checked);
-                    }}
-                  />
-                  {viewOptions.sentiment && determineToneIcon(item.tonePools[0])}
-                  {showDate && (
-                    <div className="date">{moment(item.publishedOn).format('DD-MMM-YYYY')}</div>
-                  )}
-                  <button className="headline">{item.headline}</button>
-                  <Show visible={viewOptions.section}>
-                    {item.section && <div className="section">{item.section}</div>}
-                    {item.page && <div className="page-number">{item.page}</div>}
-                  </Show>
-                  <Show visible={styleOnSettings && popOutIds?.includes(String(item.mediaTypeId))}>
-                    <FaSquareUpRight
-                      className={`new-tab ${!item.section && 'no-section'}`}
-                      onClick={(e) => {
-                        e.stopPropagation();
-                        window.open(`/view/${item.id}`, '_blank');
-                      }}
-                    />
-                  </Show>
-                  <Show
-                    visible={
-                      !!item.fileReferences.length &&
-                      !activeStream?.source &&
-                      !item.fileReferences[0].contentType.includes('image')
+      <Show visible={!handleDrop}>
+        {Object.keys(grouped).map((group) => (
+          <div key={group}>
+            <h2 className="group-title">{group}</h2>
+            <div>
+              {grouped[group].map((item) => (
+                <ContentRow
```

Comment:
Move bulk of elements to `ContentRow` for a cleaner conditional render when determining if draggable or not